recipe_scrapers/plugins/exception_handling.py

```diff
@@ -39,13 +28,17 @@ class ExceptionHandlingPlugin:
     def run(cls, decorated):
         @functools.wraps(decorated)
         def decorated_method_wrapper(self, *args, **kwargs):
-            if self.exception_handling:
+            from recipe_scrapers.settings import settings
+
+            if self.exception_handling or settings.EXCEPTION_HANDLING:
                 try:
                     return decorated(self, *args, **kwargs)
                 except Exception as e:
```

Comment:
Do you want to a new "base" exception for this package? Something like `RecipeScrapersBaseException` that all other exceptions used by this package then inherit? That way we can abstract common exceptions into something consistent? I know this would be a pattern used by other languages, but I've seen some python packages do this too.

I'm willing to start writing that in another PR.

Reply:
that's def a pattern in Python too. We do have 2 or 3 package-specific exceptions so far. I'd say adding `RecipeScrapersBaseException` is not on my most-wanted additions at this moment so I won't bother with it. However, it's def an addition I'd 👍 if you decide to do it :)